systems/node/notify/internal/config.go

```diff
@@ -27,23 +27,21 @@ type Config struct {
 	OrgName          string
 }
 
+
+
 func NewConfig(name string) *Config {
 	return &Config{
 		DB: &uconf.Database{
 			DbName: name,
 		},
-
 		Service: uconf.LoadServiceHostConfig(name),
-
 		MsgClient: &uconf.MsgClient{
-			Host:    "msg-client-notification:9095",
 			Timeout: 5 * time.Second,
 			ListenerRoutes: []string{
-				"event.cloud.global.{{ .Org}}.nucleus.org.notification.sent",
-				"event.cloud.global.{{ .Org}}.nucleus.user.notification.sent",
-				"event.cloud.local.{{ .Org}}.registry.network.notification.sent",
-				"event.cloud.local.{{ .Org}}.registry.node.notification.sent",
+				"event.cloud.local.{{ .Org}}.messaging.mesh.node.online",
+				"event.cloud.local.{{ .Org}}.messaging.mesh.node.offline",
+				"event.cloud.local.{{ .Org}}.registry.node.node.create",
```

Comment:
But we are keeping some routes ?

Reply:
The routes have been updated. Remember we had a chat with Vishal and decided to only have node related routes which were these four.

Reply:
If that''s the case, then you should start the event server. You are no long starting it, right now.